docs/guide/explanation/the-deployment-process.mdx

```diff
@@ -0,0 +1,53 @@
+# The deployment process
+Deployment begins via one of Zephyr Cloud's bundler plugins.
```

Comment:
The deployment is  more of a side effect of the build.

Reply:
Did some work on rephrasing this in [4b97ebc](https://github.com/ZephyrCloudIO/zephyr-documentation/pull/5/commits/4b97ebc4b5a7faa4c146688380a4e6dc4b217796). Let me know your thoughts.